content/src/fieldConfig/profile.json

```diff
@@ -207,7 +207,8 @@
           "headerContextualInfo": "naics",
           "editText": "Edit",
           "addText": "Add",
-          "notEnteredText": "Not entered"
+          "notEnteredText": "Not entered",
+          "altDescription": "Save your 6-digit NAICS code here so you can reference it later:"
```

Comment:
I don't think we should move, _"Save your 6-digit NAICS code here so you can reference it later:"_ in `profile.json`. My understanding is that the copy in `profile.json` is copy that lives in the **Business Profile** section of the site.